common/council_tasks/00_steward_tasks.txt

```diff
@@ -597,6 +597,18 @@ task_promote_culture = {
 				# desc = STEWARD_PROMOTE_CULTURE_INNOVATION_BONUS_EAST_SETTLING
 			# }
 		# }
+		# Warcraft
+		if = {
+			limit = {
+				scope:councillor_liege.culture = {
+					has_innovation = innovation_epidemic
+				}
+			}
+			add = {
+				value = scope:councillor.promote_culture_epidemic_bonus
+				desc = STEWARD_PROMOTE_CULTURE_INNOVATION_BONUS_EPIDEMIC
+			}
+		}
```

Comment:
Looks like it doesn't work.
Compared to the vanilla, it doesn't give a bonus.
Also, it's not localized.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/102716775-6263e000-42f7-11eb-9c24-d40c2fb5d4a0.png)
![image](https://user-images.githubusercontent.com/46384992/102716867-ed44da80-42f7-11eb-8d8b-fee5fddc7801.png)

</details>